active-fedora.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |s|
   s.add_dependency "active-triples", '~> 0.11.0'
   s.add_dependency "deprecation"
   s.add_dependency "ldp", '~> 0.6.0'
+  s.add_dependency "ruby-progressbar", '~> 1.0'
```

Comment:
Thanks for review and approval!

Personally I'd be very reluctant to create another layer of abstraction with "hydra-progressbar" or whatever, I don't understand what it gains, ruby-progressbar is already a fine API.  Excessive abstraction/adapter layers can make the code a lot less legible, and in my experience paradoxically less maintainable.